### PR TITLE
Fix Travis caching for Pyensembl and pip dirs, move Pyensembl install.

### PR DESCRIPTION
**Pull request by julia326**

### CODE REVIEW

File: .travis.yml [hunk 1]
```diff
@@ -10,11 +10,12 @@ addons:
       # Needed for NetMHC
       - tcsh
 cache:
-  pip: true
-  # cache directory used for Ensembl downloads of GTF and FASTA files
+  # cache pip files, also directory used for Ensembl downloads of GTF and FASTA files
```
Comment (tavinathanson) [inline .travis.yml:13]: What was wrong with `pip: true`?

Comment (julia326) [reply .travis.yml:13]: I don't see anything like that in the current documentation at https://docs.travis-ci.com/user/caching/, figured we should be consistent

Comment (tavinathanson) [reply .travis.yml:13]: https://docs.travis-ci.com/user/caching/#pip-cache looks similar?

Comment (julia326) [reply .travis.yml:13]: I don't know if you can have multiple `cache` entries - maybe you can? I was mostly looking at "Caching arbitrary directories" and an example listed in https://docs.travis-ci.com/user/caching/#before_cache-phase

Comment (iskandr) [reply .travis.yml:13]: Based on this random repo I think you can use multiple `cache` entries: https://github.com/travis-ci/travis-ci/issues/3239

**Comment by iskandr** [comment]
Failed due to timeout: 
```/home/travis/.travis/job_stages: line 169:  5214 Terminated              travis_jigger $! $timeout $cmd
```

File: .travis.yml (hunk 1 above)
Comment (julia326) [reply .travis.yml:13]: looks like that hasn't been updated since March of this year, and we may as well use the most recent Travis docs?

**Review by iskandr — APPROVED** [review]

**Pull request merged**